jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -397,6 +397,21 @@ void java_bytecode_convert_method_lazy(
     method_symbol.type.set(ID_C_must_not_throw, true);
   }
 
+  // Assign names to parameters
+  java_method_typet &method_type = to_java_method_type(method_symbol.type);
+  method_type.set_is_final(m.is_final);
+  java_method_typet::parameterst &parameters = method_type.parameters();
+  java_bytecode_convert_methodt::method_offsett slots_for_parameters =
+    java_method_parameter_slots(method_type);
+  expanding_vectort<std::vector<java_bytecode_convert_methodt::variablet>>
+    dummy_local_variable_table;
+  construct_parameter_names(
+    m,
+    method_identifier,
+    parameters,
+    slots_for_parameters,
+    dummy_local_variable_table);
```

Comment:
Use a non-owning pointer (i.e. `expanding_vectort<std::vector<java_bytecode_convert_methodt::variablet>> *`) rather than a dummy. Or, factor the method a little more (`create_parameter_local_variables` for example).

Reply:
Turns out to be unnecessary at the end, so removed.